docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  db:
+    image: postgres:16.4
+    restart: unless-stopped
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 3s
+      timeout: 3s
+      retries: 5
+    env_file: ./server/.env
```

Comment:
There's a comment in the server .env.example that needs to be removed because it's outside the dev container network now

Reply:
Which one?